indico/modules/categories/templates/management/content.html

```diff
@@ -8,7 +8,8 @@
             <a class="i-button-icon icon-handle"></a>
         </td>
         <td class="i-table column-icon">
-            <input type="checkbox" name="category_id" value="{{ subcategory.id }}">
+            <input type="checkbox" name="category_id" value="{{ subcategory.id }}"
+                   data-is-empty="{{ subcategory.is_empty|tojson }}">
```

Comment:
While it doesn't matter for a boolean I'd always use `|forceescape` after `|tojson` when it's used outside script tags